src/styles/_variables.scss

```
@@ -43,6 +43,9 @@ $results-inline-gutter: clamp(
 );
 $header-height: 2.75rem;
 
+$modal-width: 36rem;
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)